EpikCord/managers/cache_manager.py

```diff
@@ -1,11 +1,16 @@
+import asyncio
 from typing import Any, Optional
 
 
 class CacheManager:
-    def __init__(self):
+    def __init__(self, limit= 5000):
```

Comment:
5000 is a very odd number, i wonder if we wouldn't be better having no limit by default and let the user and one it they feel necessary